src/resources/views/list.blade.php

```diff
@@ -43,8 +45,9 @@
                   <th
                     data-orderable="{{ var_export($column['orderable'], true) }}"
                     data-priority="{{ $column['priority'] }}"
+                    data-visible-in-modal="{{ (isset($column['visibleInModal']) && $column['visibleInModal'] == false) ? 'false' : 'true' }}"
                     >
-                    {{ $column['label'] }}
+                    {!! $column['label'] !!}
```

Comment:
I'm still new to using Backpack, but why was this changed? You can use `\Illuminate\Support\HtmlString` as the return value of `$column['label']` whenever you need HTML and keep using `{{ }}` for safe filtering of output. Same for line 62 below.

Reply:
It allows developers to put HTML inside the label. It's because we wanted general checkboxes in the table header, so we needed to be able to do this:
```php
$this->crud->addColumn([
            'type' => 'checkbox',
            'name' => 'bulk_actions',
            'label' => '<input type="checkbox" />',
        ]);
```

but now since we've opened the door, other developers could also use it for things like
```php
$this->crud->addColumn([
            'type' => 'select',
            'name' => 'tags',
            'label' => 'Tags <i class="fa fa-tags" data-toggle="tooltip" title="Tags that have been associated with this entry."></i>',
        ]);
```

I remember someone saying a while back that they needed this. At the time I disagreed too - I didn't see how/why you'd need to put HTML inside the label. So back then I totally agreed with you. But now, apparently, I have to track that person down to tell them they were right :-)

BUT.

The escaping is only changed for the ```label```, which is defined by the developer, never the user. Since it's something _the developer_ defines, I think the extra security is for nothing, but the extra flexibility is... for something :-))

If the developer wants to output HTML there, I say let him have it.

Reply:
It would be just as easy for the developer to do something like this instead:

```
$this->crud->addColumn([
    'type' => 'select',
    'name' => 'tags',
    'label' => new HtmlString('Tags <i class="fa fa-tags" data-toggle="tooltip" title="Tags that have been associated with this entry."></i>'),
]);
```

but if you want to keep it this way, that's fine. I was just wondering why the change. Thank you for answering. :)